### DISCUSSION TITLE
Qualcomm Atheros AR3012 Not working (Integrated laptop BT adapter)

### DISCUSSION THREAD

**Discussion by AshIndigo** [nefarius/BthPS3#16] [Q&A]
I just installed this to try and get BT working with my controller, but I'm getting this error. Is it just not compatible and I'm out of luck or is there something I can do?

> This device cannot start. (Code 10)
> 
> STATUS_DEVICE_POWER_FAILURE

**Comment by nefarius** [comment]
Please share the Hardware ID and HCI/LMP version (see README and [here](https://vigem.org/projects/BthPS3/Compatible-Bluetooth-Devices/)).

**Comment by AshIndigo** [comment]
HCI 6.4354 / LMP 6.1
Hardware ID: USB\VID_0CF3&PID_3004 and USB\VID_0CF3&PID_3004&REV_0001

**Comment by nefarius** [comment]
[Please exercise this](https://vigem.org/projects/BthPS3/Debugging-the-drivers/) by disabling your Bluetooth adapter, start the trace, enable it so the error gets captured, stop the trace and attach it here.

**Comment by AshIndigo** [comment]
[BthPS3.zip](https://github.com/ViGEm/BthPS3/files/6130556/BthPS3.zip)

Sorry for the delay, I've attached the etl file inside of the zip

**Comment by nefarius** [comment]
No worries. Interesting, never seen before 🤔 

```
 [BthPS3_RegisterL2CAPServer] ERROR:BRB_REGISTER_PSM failed with status STATUS_DEVICE_BUSY (0x80000011)
```

Does it work if you if you disable and enable the `Nefarius Bluetooth PS Enumerator` device?

![image](https://user-images.githubusercontent.com/286631/110971862-aba65780-835b-11eb-8b90-2502a45b3db6.png)

**Comment by AshIndigo** [comment]
Just toggled it on and off by disabling and enabling, and still got the same message as before.

**Comment by nefarius** [comment]
Damn, uh, good question then 😬

**Comment by nefarius** [comment]
Yeah mate I'm afraid I have no idea on this one...

**Comment by MatadorMaster33** [comment]
device not started(bthPS3)
The BTHENUM device \ {1cb831ea-79cd-4508-b0fc-85f7c85ae8e0} _LOCALMFG & 0000 \ 8 & 1f343510 & 0 & 000000000000_00000002 had a problem starting.

Driver name: oem19.inf
Class GUID: {e0cbf06c-cd8b-4647-bb8a-263b43f0f974}
Service: BthPS3
Bottom filters:
Top filters:
Problem: 0x1F
Problem status: 0xC0000034

**Reply by nefarius** [reply]
I don't know and can't offer a solution.

**Comment by MatadorMaster33** [comment]
![image](https://user-images.githubusercontent.com/88615524/129800015-7b8cdc5b-6738-434e-8648-8122a67739eb.png)
I found this problem, it happens that my ps3 controller is failing the wireless connection

**Reply by nefarius** [reply]
And what does that have to do with this discussion? Why did you hijack it? Has literally nothing to do with it? And Shibari is end of life BTW.
